module/gui/Overview.py

```diff
@@ -141,30 +147,29 @@ def paint(self, painter, option, index):
         status = index.data(OverviewModel.Status).toString()
         
         def formatEta(seconds): #TODO add to utils
-            if seconds <= 0: return ""
+            if seconds <= 0: return unicode("")
             hours, seconds = divmod(seconds, 3600)
             minutes, seconds = divmod(seconds, 60)
-            return _("ETA: ") + "%.2i:%.2i:%.2i" % (hours, minutes, seconds)
+            return _("ETA") + ": %.2i:%.2i:%.2i" % (hours, minutes, seconds)
         
-        statusline = QString(_("Parts: ") + "%s/%s" % (partsf, parts))
-        if partsf == parts:
-            speedline = _("Finished")
-        elif not status == _("Downloading"):
-            speedline = QString(status)
+        statusline = QString(_("Parts") + ": %s/%s" % (partsf, parts))
+        if status == _("downloading"):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)